service/matching/taskQueueManager_test.go

```diff
@@ -847,16 +709,16 @@ func TestMutateOnNonRootFails(t *testing.T) {
 	tqCfg := defaultTqmTestOpts(controller)
 	tqCfg.tqId = subTqId
 	subTq := mustCreateTestTaskQueueManagerWithConfig(t, controller, tqCfg)
+	subTq.dbTaskManager.Start()
 	err = subTq.MutateVersioningData(ctx, func(data *persistencespb.VersioningData) error { return nil })
 	require.Error(t, err)
-	require.ErrorIs(t, err, errVersioningDataNoMutateNonRoot)
```

Comment:
This error type is pretty important to verify, why remove it?

Same thing a few lines later.

Reply:
Removed by mistake during rebase. Added back.